src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -234,13 +234,13 @@ ul.sub-nav-list li {
 .sub-title {
     margin:5px 0 0 0;
 }
-.header ul {
-    margin:0 0 15px 0;
-    padding:0;
+ul.contents-list {
+    margin: 0 0 15px 0;
+    padding: 0;
+    list-style: none;
 }
-.header ul li {
-    list-style:none;
-    font-size:13px;
+ul.contents-list li {
+    font-size: 13px;
```

Comment:
Do these values have to be normalized into `.em` units, to align with new themes?